Lib/importlib/abc.py

```diff
@@ -13,7 +13,6 @@
     _frozen_importlib_external = _bootstrap_external
 from ._abc import Loader
 import abc
-import warnings
```

Comment:
This import is used on this branch, see https://github.com/python/cpython/issues/121279